helm/cluster/values.schema.json

```diff
@@ -1745,7 +1768,30 @@
                                     "description": "Advanced configuration of controller manager",
                                     "properties": {
                                         "featureGates": {
-                                            "$ref": "#/$defs/featureGates"
+                                            "type": "array",
+                                            "title": "Feature Gates",
+                                            "description": "List of feature gates on every clusters.",
+                                            "items": {
+                                                "type": "object",
+                                                "properties": {
+                                                    "enabled": {
+                                                        "type": "boolean",
+                                                        "title": "Enabled",
+                                                        "description": "Enable the corresponding feature gate."
+                                                    },
+                                                    "name": {
+                                                        "type": "string",
+                                                        "title": "Name",
+                                                        "description": "Name of the feature gate."
+                                                    }
+                                                }
+                                            },
```

Comment:
```suggestion
                                            "$ref": "#/$defs/featureGates",
```

Reply:
Why shouldn't we have the full description ? According to what is done on the other parts of the schema, it shouldn't juste have a `$ref` right ?

Reply:
Because $ref is already referencing a schema definition with description and everything. This is where the docs are being generated from. Also that template is being used in other places. If you're not fine with the description of the definition, please update it there. But we'd like to use those definitions where possible to reduce code duplication.

Reply:
Ok got it. Replaced it as you suggested :)